docker-images/base/Dockerfile

```diff
@@ -5,7 +5,7 @@ ARG TARGETPLATFORM
 USER root
 
 RUN microdnf update \
-    && microdnf install java-${JAVA_VERSION}-openjdk-headless openssl procps shadow-utils \
+    && microdnf install java-${JAVA_VERSION}-openjdk-headless java-${JAVA_VERSION}-openjdk-devel openssl procps shadow-utils \
```

Comment:
Why do we need this?

Reply:
This is only to support collection of JVM thread dumps in my testing. The final PR will not have any changes to `Dockerfile`.